src/containerapp/azext_containerapp/custom.py

```diff
@@ -1240,6 +1244,73 @@ def list_certificates(cmd, name, resource_group_name, location=None, certificate
     return list_certificates_logic(cmd, name, resource_group_name, location, certificate, thumbprint, managed_certificates_only=managed_certificates_only, private_key_certificates_only=private_key_certificates_only)
 
 
+def upload_certificate(cmd, name, resource_group_name, certificate_file=None, certificate_name=None, certificate_password=None, location=None, prompt=False, certificate_identity = None, certificate_key_vault_url=None):
```

Comment:
Recommend to use decorator to rewrite upload certificate.

Reply:
Command `env certificate list` also needs to overwrite in the extension, because the new feature contains new properties `certificateKeyVaultProperties`. After rewriting it in the containerapp extension, then it can uses api-version `2022-11-02-preview ` to list.